<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogChCell</class>
 <widget class="QDialog" name="DialogChCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="SelectChannel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>103</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SelectCell">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1023</number>
   </property>
   <property name="value">
    <number>512</number>
   </property>
  </widget>
  <widget class="QLabel" name="LabelSelectCh">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelectSel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Cell</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SelectNbEventsEnd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelNbEvents">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nb events fin</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ListFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>361</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Choice File</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SelectNbEventsBegin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelNbEvents_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nb events debut</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogChCell</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogChCell</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
